plugins/it.xsemantics.dsl/NEWS.txt

```diff
@@ -0,0 +1,15 @@
+*******************************************************************************
```

Comment:
This file is pretty old and is not kept up to date... I'd take the chance and remove it :)